src/test/java/org/myrobotlab/service/SolrTest.java

```diff
@@ -114,7 +115,7 @@ public void testService() throws Exception {
     
     Assert.assertEquals(1, resp.getResults().getNumFound());
     
-    ProgramAB.Response programABResponse = new ProgramAB.Response("joe", "lloyd", "this is a test response.", null, new Date());
+    Response programABResponse = new Response("joe", "lloyd", "this is a test response.", null);

```

Comment:
What happened to the date object in the constructor here?  This shouldn't change.

Reply:
its an appropriate internal timestamp, as it should be